src/shared/components/common/password-input.tsx

```diff
@@ -0,0 +1,153 @@
+import { Options, passwordStrength } from "check-password-strength";
+import classNames from "classnames";
+import { NoOptionI18nKeys } from "i18next";
+import { Component, FormEventHandler, linkEvent } from "inferno";
+import { NavLink } from "inferno-router";
+import { I18NextService } from "../../services";
+import { Icon } from "./icon";
+
+interface PasswordInputProps {
+  id: string;
+  value?: string;
+  onInput: FormEventHandler<HTMLInputElement>;
+  className?: string;
+  showStrength?: boolean;
+  label?: string | null;
```

Comment:
If you have `?`, I don't think `| null` is necessary.

Reply:
The `i18n.t` function returns a null if the translation doesn't exist. That's why there were so many null coalesces before.